### ISSUE TITLE
Notify users of auto-import or make it more obvious (ES6)

### ISSUE THREAD

**Issue by alxbradley** [microsoft/vscode#107994]
Issue Type: <b>Feature Request</b>

I accidentally imported a package after typing a function name that exists in a package that isn't part of our build. The webpack build started failing but I didn't notice right away.

The import was added to the very top of the file above our license comment which made it difficult to see. I spent 30+ minutes trying to find what could possibly cause the issue.

I'm not actually sure what key sequence I made to auto-import that package, but it must have been a typo on my part.

VS Code version: Code 1.49.2 (e5e9e69aed6e1984f7499b7af85b3d05f9a6883a, 2020-09-24T16:29:41.983Z)
OS version: Windows_NT x64 10.0.18362

**Comment by mjbvz** [comment]
Auto imports are added when you accept a suggestion. Here's an example of what we show for them: 

<img width="1163" alt="Screen Shot 2020-10-02 at 1 02 23 PM" src="https://user-images.githubusercontent.com/12821956/94965014-90cbdf00-04af-11eb-9b5e-587c8e2c4a11.png">

They should always start with `Auto import`. The added import will also show in source control if you are using that

Any suggestions on making them more obvious?

**Comment by alxbradley** [comment]
A few ideas:
1. A confirmation banner that gets focus. It could use the same UI element as the command palette. "Are you sure you want to add `foobar` package" (Yes/No)?
2. Place auto imports below the first comment if a comment exists at the top of the file.

**Comment by SirCameron** [comment]
I have a similar issue. Vscode is auto importing packages that aren't even a part if the current project.
What's the point of that? type 'expect' hit enter and it imports Chai.
Am I doing something stupid?

**Comment by mjbvz** [comment]
The import being added above the comment sounds like a bug. Can you please open a separate issue for that with some example code

Regarding a notification: we've had auto imports for something like three years now and have not see other requests for making them more obvious. I am very hesitant to introduce confirmations or notifications or anything else that could slow user down or make the editor feel like it's getting in the way of the programmer. 

If there's a specific case where you feel auto imports are being incorrectly added, it may be a configuration issue with your project. If you've tried that but still are running into problems, open another issue with a small example and we can investigate further

**Comment by alxbradley** [comment]
@mjbvz I think the main issue is the context menu is _too fast_ in some cases.

Whatever I typed brought up the menu faster than I realized and I had already pressed `Enter` or `Tab` without seeing what change I was accepting. That's great when you're typing something and you know exactly which item is going to be selected before-hand, but if it's something unexpected, then you'll run into the problem I had.

By the way, this doesn't only happen for auto-imports, this happens to me for other auto-complete options as well. Since the auto-complete actions replace text in-line it's obvious that you did something wrong. You will _see_ the change immediately. In the case of the auto-import, it wasn't obvious and hard to find what I accepted.

I shouldn't have to stop and read the context menu. I know it's a tricky UX problem. Of course we want things to be fast, but it also shouldn't interfere with what I'm typing.
